libminifi/test/rocksdb-tests/DBContentRepositoryTests.cpp

```diff
@@ -140,7 +140,7 @@ TEST_CASE("Test Empty Claim", "[TestDBCR3]") {
   std::string readstr;
 
   // -1 tell us we have an invalid stream
```

Comment:
minor, but you changed "-1" to "error" in the other place -- should be changed here, too

Reply:
changed in 46f3b0e